content/07.pathogenesis.md

```diff
@@ -9,6 +9,11 @@ This section would also be great for the introduction of zoonotic diseases which
 [How does SARS-CoV-2 enter human cells?]
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
+ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145; @doi:10.1007/s00134-020-05985-9; @doi:10.1128/JVI.00127-20]. 
+The virus invades host cells by using ACE2 for entry and the serine protease TMPRSS2 for S protein priming. 
+It then fuses with the membrane and enters the cell via endosomes [@doi:10.1016/j.cell.2020.02.052]. 
+The C-terminal domain of the SARS-CoV-2 S protein was identified as the key region of the virus that interacts with the human ACE2 (hACE2) receptor. 
+The crystal structure of the C-terminal domain of SARS-CoV-2 S protein in complex with human ACE2 reveals stronger interaction and higher affinity for receptor binding than SARS-CoV [doi:10.1016/j.cell.2020.03.045]. 
```

Comment:
I think we should be a bit careful when claiming that SARS-CoV-2 S RBD can bind hACE2 more strongly than can SARS-CoV S RBD. The paper you cited as well as [@doi:10.1126/science.abb2507] found stronger binding by SARS-CoV-2, but two other recent papers found comparable binding [@doi:10.1016/j.cell.2020.02.058] [@doi:10.1101/2020.02.19.956235]...